src/test/java/org/fundacionjala/pivotal/steps/ProjectStep.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.pivotal.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.pivotal.objects.CreateProject;
+import org.fundacionjala.pivotal.objects.DashBoard;
+import org.fundacionjala.pivotal.objects.ProjectValues;
+import org.junit.Assert;
+
+import java.util.Map;
+
+/**
+ * Class Project Steps for create a project
+ */
+public class ProjectStep {
+    private CreateProject createProject = new CreateProject();
+    private Map<ProjectValues, Object> projects;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "projects" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)